src/common/updating/updating.cpp

```diff
@@ -8,11 +8,14 @@
 
 #include <common/SettingsAPI/settings_helpers.h>
 #include <common/utils/json.h>
+#include <common/utils/registry.h>
+
+using namespace registry::install_scope;
 
 namespace // Strings in this namespace should not be localized
 {
-    const wchar_t LATEST_RELEASE_ENDPOINT[] = L"https://api.github.com/repos/microsoft/PowerToys/releases/latest";
-    const wchar_t ALL_RELEASES_ENDPOINT[] = L"https://api.github.com/repos/microsoft/PowerToys/releases";
+    const wchar_t LATEST_RELEASE_ENDPOINT[] = L"https://api.github.com/repos/stefansjfw/PowerToys/releases/latest";
+    const wchar_t ALL_RELEASES_ENDPOINT[] = L"https://api.github.com/repos/stefansjfw/PowerToys/releases";
```

Comment:
Requesting a change here so we don't forget to rollback the endpoints to actual production endpoints.